source/index.html.md

```diff
@@ -1,239 +1,504 @@
+
+Supports `gzip` and `zip` compression from levels 1-9. Specify parameters this way:
+
+<code>...&compress_with=gzip-9</code>
+
+Default compression level is 6 and default compression type `zip`.
+
+# Aggretation
+
+Use aggregation functions to retrieve data set-level information
+
+## <span class="get">GET</span> Aggregate
+
+>https://api.namara.io/v0/data_sets/:data_set_id/data/:version/aggregation
+
+### Request
+
+Parameter | Type | Description
+--------- | ---- | -----------
+data_set_id (required) | `string` | UUID for accessing the data set
+version (required) | `string` | Version identifier, eg. `en-0`
+operation (required) | `string` | Operation function to perform (see <a href="#operations">Operations</a>)
+where | `string` | Conditions for performing query (see <a href="#conditions">Conditions</a>)
+
+<aside class="notice">The only optional parameter here is the `where` - all others are mandatory</aside>
+
+## Response
+
+Response | Description
+-------- | -----------
+200: OK |
+
+## Operations
+
+Function | Description | Use
+-------- | ----------- | ---
+count | The number of rows of data. If a property key is specified instead of *, this will only count the rows where the given property is not null. | `count(*)` `count(p0)`
+sum | The sum of all values in a column | `sum(p0)`
+avg | The average of all values in a column | `avg(p0)`
+min | The minimum value in a column | `min(p0)`
+max | The maximum value in a column | `max(p0)`
+
+## Operator Examples
+
+>1) <code>https://api.namara.io/v0/data_sets/057d7914-839e-4625-b8f8-2aa109f11e5a/data/en-3/aggregation?api_key={YOUR_API_KEY}&operation=avg(co2_emissions_g_km)&where=make="CADILLAC"</code>
+
+>2) <code>https://api.namara.io/v0/data_sets/057d7914-839e-4625-b8f8-2aa109f11e5a/data/en-3/aggregation?api_key={YOUR_API_KEY}&operation=min(co2_emissions_g_km)&where=make="CADILLAC"</code>
+
+>3) <code>https://api.namara.io/v0/data_sets/057d7914-839e-4625-b8f8-2aa109f11e5a/data/en-3/aggregation?api_key={YOUR_API_KEY}&operation=count(*)&where=make="CADILLAC"</code>
+
+1. Reveals the average CO<sub>2</sub> emissions of Cadillac vehicles
+
+2. Reveals the Cadillac vehicle with the least CO<sub>2</sub> emission
+
+3. Reveals the Cadillac vehicles that have emissions data
+
+# Query API
+
+While the Data API allows query behaviour over a single data set, the Query API allow broader control over the Namara data catalog.
+
+This API supports NiQL (the Namara io Query Language), a language for viewing, aggregating, and joining data sets. It's very similar to SQL, so don't worry - no new textbooks.
+
+##<span class="get">GET</span> Meta
+
+>https://api.namara.io/v0/query/meta
+
+This endpoint provides the meta information for querying this Namara instance. It includes the maximum limit per query, the supported formats, and the default format if none is specified. This information may differ, depending on which deployment of Namara you're using. 
+
+*then maybe a chart? req res?*
```

Comment:
The keys will stay the same, the values may differ depending on namara instance. For now the values will be:

```
{
  "query_limit_maximum": 250,
  "formats: ["geojson", "csv", "json"],
  "default_format": "json"
}
```